src/Symfony/Bridge/PhpUnit/FailTests/ExpectDeprecationTraitFailTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\FailTests;
```

Comment:
As you create a new folder for some testsuite, it must be added in `.gitattribute` (or it should be kept inside the `Tests` folder to avoid creating a new convention)

Reply:
I put it here to avoid PHPUnit finding it. But I worked out a better way so that it can be in the Tests directory.